frontend/src/components/AdminComponents/PatientBox.js

```diff
@@ -0,0 +1,133 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+import styles from "./PatientBox.module.css";
+import { useState, useEffect } from "react";
+
+import { Accordion } from "react-bootstrap";
+import { Button } from "react-bootstrap";
+import AccordionBody from "react-bootstrap/esm/AccordionBody";
+import { Modal } from "react-bootstrap";
+import { ListGroup } from "react-bootstrap";
+
+import axios from "axios";
+
+const eventKey = "0";
+const patient = null;
+const currentUser = null;
+let selectedDoctorID = "";
+let selectedDoctorName = "";
+
+function PatientBox(props) {
+  
+  const [show, setShow] = useState(false);
+
+  const [doctorList, setDoctorList] = useState([]);
+
+  const [assignedDoctor, setAssignedDoctor] = useState(props.patient.patientInfo.doctor);
+  
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  const openDoctorList = () => {
+    handleShow();
+    fetchDoctorList();
+  }
+
+  const selectDoctor = (id, name) => {
+    selectedDoctorID=id;
```

Comment:
When selecting a doctor in the UI, it is not clear if the doctor was selected or not. Maybe add check boxes to make it clear?